test/e2e/persistent_volumes.go

```diff
@@ -41,18 +42,17 @@ func completeTest(f *framework.Framework, c clientset.Interface, ns string, pv *
 	By("Checking pod has write access to PersistentVolume")
 	createWaitAndDeletePod(f, c, ns, pvc.Name)
 
-	// 3. delete the PVC, wait for PV to become "Available"
+	// 3. delete the PVC, wait for PV to become "Released"
```

Comment:
`completeTest` and `completeMultiTest` still both mention "triggering the recycler" even though that only occurs in a single Context in _persistent_volumes.go_